runelite-client/src/main/java/net/runelite/client/chat/CommandManager.java

```diff
@@ -125,7 +127,7 @@ private void handleInput(ScriptCallbackEvent event)
 		int stringStackCount = client.getStringStackSize();
 		int intStackCount = client.getIntStackSize();
 
-		final String typedText = stringStack[stringStackCount - 1];
+		String typedText = stringStack[stringStackCount - 1];
```

Comment:
Why was this file changed?

Reply:
Line 146. Otherwise, you can't send updated Chatbox value.